src/Knub.ts

```diff
@@ -181,9 +181,13 @@ export class Knub<
       this.emit("loadingFinished");
     });
 
-    this.client.ws.on("GUILD_CREATE", (data: { id: string }) => {
+    this.client.ws.on("GUILD_CREATE", (data: any) => {
```

Comment:
You could use the [`discord-api-types`](https://npmjs.com/package/discord-api-types) library (which was made by the discord.js team) to strong type this data

Reply:
True, true. Thank you!